Update __init__.py

fzarifian opened the pull request:
#4304

Thanks for submitting a PR, your contribution is really appreciated!

Here's a quick checklist that should be present in PRs (you can delete this text from the final description, this is
just a guideline):

- [ ] Create a new changelog file in the `changelog` folder, with a name like `<ISSUE NUMBER>.<TYPE>.rst`. See [changelog/README.rst](https://github.com/pytest-dev/pytest/blob/master/changelog/README.rst) for details.
- [ ] Target the `master` branch for bug fixes, documentation updates and trivial changes.
- [ ] Target the `features` branch for new features and removals/deprecations.
- [ ] Include documentation when adding new features.
- [ ] Include new tests or update existing tests when applicable.

Unless your change is trivial or a small documentation fix (e.g.,  a typo or reword of a small section) please:

- [ ] Add yourself to `AUTHORS` in alphabetical order;

fzarifian committed:
Update __init__.py

#4304

RonnyPfannschmidt commented:
you might want to add a integration test and open the pr against the correct repository
i'd like to note that this change is a bit controversial as we try to avoid opt in/opt out of internal plugins

fzarifian commented:
> i'd like to note that this change is a bit controversial as we try to avoid opt in/opt out of internal plugins

I don't really know where you want to fix it, by the way i don't have time to find the correct way to do it...

fzarifian committed:
Add test to branch

fzarifian committed:
Update __init__.py

fzarifian committed:
Create 4304.bugfix.rst

fzarifian committed:
Update __init__.py

As mentionned by @RonnyPfannschmidt, use set_blocked on module stepwise (and it's brother pytest_stepwise like consider_pluginarg method does)

nicoddemus committed:
Improve changelog a bit

nicoddemus committed:
Fix linting

fzarifian committed:
Update AUTHORS
